examples/.cargo/config.toml

```diff
@@ -28,6 +28,11 @@ rustflags = [
 
 [env]
 ESP_LOGLEVEL = "info"
+SSID = "SSID"
```

Comment:
I just added this because otherwise you can't run `cargo xtask build-examples esp-hal $CHIP` locally without it failing, which got really annoying really quick when I was trying to verify my changes. Can remove this if you think I should, though.

Reply:
No, no! I like that ... env-vars set by the user will override them 👍 

If it ever causes problems for someone we can check for the defaults in the examples at runtime and give a hint. But _if_ everyone reads the comments of the examples it should never be a problem